libraries/AP_Math/AP_Math.cpp

```diff
@@ -28,7 +28,7 @@ is_equal(const Arithmetic1 v_1, const Arithmetic2 v_2)
     if (std::is_same<double, common_type_nonconst>::value) {
         return fabs(v_1 - v_2) < std::numeric_limits<double>::epsilon();
     }
-    return fabsf(v_1 - v_2) < std::numeric_limits<float>::epsilon();
+    return std::abs(v_1 - v_2) < std::numeric_limits<float>::epsilon();
```

Comment:
std::abs should work the same that fabsf, I don't understand why clang trigger a warning. I case of double it should reach this, no ?